app/src/main/java/de/danoeh/antennapod/activity/OnlineFeedViewActivity.java

```diff
@@ -476,8 +476,8 @@ private void showFeedInformation(final Feed feed, Map<String, String> alternateF
             for (String url : alternateFeedUrls.keySet()) {
                 alternateUrlsTitleList.add(alternateFeedUrls.get(url));
             }
-            ArrayAdapter<String> adapter = new ArrayAdapter<>(this, android.R.layout.simple_spinner_item, alternateUrlsTitleList);
-            adapter.setDropDownViewResource(android.R.layout.simple_spinner_dropdown_item);
+            ArrayAdapter<String> adapter = new ArrayAdapter<>(this, R.layout.alternate_urls_item, alternateUrlsTitleList);
```

Comment:
CI complains because the line is too long. I'll just fix it here.

```suggestion
            ArrayAdapter<String> adapter = new ArrayAdapter<>(this,
                    R.layout.alternate_urls_item, alternateUrlsTitleList);
```